src/map.rs

```diff
@@ -2,11 +2,18 @@ use crate::iter::*;
 use crate::node::*;
 use crate::raw::*;
 use crossbeam_epoch::{self as epoch, Atomic, Guard, Owned, Shared};
+#[cfg(feature = "serialize")]
```

Comment:
If we remove the feature and just use `optional` on `serde`, this would become:
```rust
#[cfg(feature = "serde")]
```

Reply:
Just fixed it.